### DISCUSSION TITLE
Best way to handle roles?

### DISCUSSION THREAD

**Discussion by Jarrodsz** [sergiodxa/remix-auth#206] [Q&A]
What would be the best way to handle roles. I found cerbos.dev to handle this but not sure, just investigating options for the near future.

Perhaps its out of scope of this project and there could be a community effort to build this feature set. 

Cerbos does look interesting since it also seem to do more but perhaos roling a build system is in the end more convient and not needing external tools is something i would prefer.

**Comment by sergiodxa** [comment]
This depends a lot in your app, the simplest way is to store the role in your DB and return it from the strategy verify callback as part of your user data. Then in your loaders and actions check the user role.

Another option could be to get the role again from the DB/API when you need it in case it's common the role may change while the user is authenticated, but I think that's not that common.
